src/grpc_interceptor/client.py

```diff
@@ -131,6 +132,32 @@ def intercept_stream_stream(
         return self.intercept(_swap_args(continuation), request_iterator, call_details)
 
 
+class AsyncClientInterceptor(
```

Comment:
nice, and then we just have to implement these and call our new intercept method?

Reply:
yes, we can refer to the tests.